css/deltagreen.css

```diff
@@ -671,11 +672,15 @@ div .bottom-left-justified-resource {
 }
 
 div .first-aid-received-section {
+  display: grid;
+  grid-template-columns: 40% 70px;
   border-bottom: 1px solid #aaa;
   margin-bottom: 10px;
 }
 
 div .exhausted-section {
+  display: grid;
```

Comment:
This is to align checkboxes from aid received and exhausted section, which are in two consecutive rows, and have different aligned checkboxes